README.md

```diff
@@ -1,61 +1,46 @@
-# Azure Functions Serverless Plugin 
+# Azure Functions Serverless Plugin
 
 This plugin enables Azure Functions support within the Serverless Framework.
 
 [![Coverage Status](https://coveralls.io/repos/github/serverless/serverless-azure-functions/badge.svg)](https://coveralls.io/github/serverless/serverless-azure-functions)
 
-## Getting started
+## Quickstart
 
 ### Pre-requisites
 
 1. Node.js v6.5.0+ *(this is the runtime version supported by Azure Functions)*
 2. Serverless CLI `v1.9.0+`. You can run `npm i -g serverless` if you don't already have it.
 3. An Azure account. If you don't already have one, you can sign up for a [free trial](https://azure.microsoft.com/en-us/free/) that includes $200 of free credit.
 
-### Create a new Azure service
-
-1. Create a new service using the standard Node.js template, specifying a unique name for your app: `serverless create -t azure-nodejs -p <appName>`
-2. CD into the generated app directory: `cd <appName>`
-3. Install the app's NPM dependencies, which includes this plugin: `npm install`
-
-### Creating or removing Azure Functions
-
-To create a new Azure Function within your function app, run the following command from within your app's directory:
+### Create a new Azure Function App
 
 ```bash
-sls func add -n {functionName}
+# Create Azure Function App from template
+$ sls create -t azure-nodejs -p <appName>
```

Comment:
The sample app created with the template has its own readme.md file which is outdated. it needs to have the same contents as this document. Could you please modify that as well

Reply:
Sure thing. That will be the next step, updating both that README as well as the quickstart docs in the Serverless core